.doc_gen/metadata/cloudwatch-logs_metadata.yaml

```diff
@@ -0,0 +1,82 @@
+# zexi 0.2.0
+cloudwatch-logs_DescribeSubscriptionFilters:
+  title: Describing &CWL; subscription filters metrics using an &AWS; SDK
+  title_abbrev: Describe subscription filters metrics
+  synopsis: describe &CWLlong; subscription filters metrics.
```

Comment:
The phrase "subscription filters metrics" reads weird to me. Maybe just "subscription filter"?
So things like: "Describe a subscription filter"